### PR TITLE
Updated: the webinar UI design

### PR DESCRIPTION
**Pull request by POBOROJO**
This pull request introduces several significant changes to enhance the webinar page, including the addition of a new hook for certificate handling, improvements to the webinar page layout, and updates to the data fetching logic.

### New Hook:
* Added a new hook `useCertificate` to manage certificate generation and download functionality (`src/hooks/useCertificate.ts`, `src/hooks/index.ts`). [[1]](diffhunk://#diff-0904a354f8f20d04c8fcd117edecd2d95931f1b7e582cf650c7489cd557ba0f0R1-R48) [[2]](diffhunk://#diff-b96dae4653c651669a3c7a95ddf01ed8823a967561b1a74951e2640b2bcbc174R3-R7)

### Webinar Page Enhancements:
* Updated the webinar page to include new sections such as date and time, registration, webinar details, learning outcomes, instructor information, and student testimonials (`src/pages/webinar/[webinarSlug]/index.tsx`). ([src/pages/webinar/[webinarSlug]/index.tsxR12-R31](diffhunk://#diff-7968a24aa67c0c9f80428ac23643a9cbcd61936558128d0294e2d6f660633255R12-R31), [src/pages/webinar/[webinarSlug]/index.tsxR254-R526](diffhunk://#diff-7968a24aa67c0c9f80428ac23643a9cbcd61936558128d0294e2d6f660633255R254-R526), [src/pages/webinar/[webinarSlug]/index.tsxL273-R538](diffhunk://#diff-7968a24aa67c0c9f80428ac23643a9cbcd61936558128d0294e2d6f660633255L273-R538), [src/pages/webinar/[webinarSlug]/index.tsxR553-R571](diffhunk://#diff-7968a24aa67c0c9f80428ac23643a9cbcd61936558128d0294e2d6f660633255R553-R571))

### Data Fetching Updates:
* Modified the `getWebinarPageProps` function to fetch additional webinar details like `aboutWebinar` and `whatYoullLearn` (`src/utils/global.ts`). [[1]](diffhunk://#diff-0e244cac02de193a0280438bc49010aa75527159b685df1cd74fcf431cdeac00R270-R271) [[2]](diffhunk://#diff-0e244cac02de193a0280438bc49010aa75527159b685df1cd74fcf431cdeac00R296-R297)

### CODE REVIEW

File: src/pages/webinar/[webinarSlug]/index.tsx [hunk 1]
```diff
@@ -52,7 +61,7 @@ const WebinarPage = ({
         link.download = `${userName}-webinar-certificate.png`;
         link.click();
       } catch (error) {
-        console.error('Error generatingcertificate image:', error);
+        console.error('Error generating certificate image:', error);
```
Comment (imsks) [inline src/pages/webinar/[webinarSlug]/index.tsx:63]: Can you convert into a Hook -> useCertificate

File: src/pages/webinar/[webinarSlug]/index.tsx [hunk 2]
```diff
@@ -137,13 +146,13 @@ const WebinarPage = ({
           Generate Your Certificate
         </Text>
 
-        <FlexContainer fullWidth={true} direction='row' className='gap-2 '>
+        <FlexContainer fullWidth={true} direction='row' className='gap-2'>
           <FlexContainer
             direction='col'
-            className='w-full md:w-[45%] max-w-[400px] '
+            className='w-full md:w-[45%] max-w-[400px]'
```
Comment (imsks) [inline src/pages/webinar/[webinarSlug]/index.tsx:151]: Avoid using Absolute values.

File: src/pages/webinar/[webinarSlug]/index.tsx [hunk 3]
```diff
@@ -137,13 +146,13 @@ const WebinarPage = ({
...
             itemCenter={false}
           >
-            <Text level='label' className='pre-name  text-white font-semibold'>
+            <Text level='label' className='pre-name text-white font-semibold'>
```
Comment (imsks) [inline src/pages/webinar/[webinarSlug]/index.tsx:154]: Remove classes

File: src/pages/webinar/[webinarSlug]/index.tsx [hunk 4]
```diff
@@ -137,13 +146,13 @@ const WebinarPage = ({
...
               Your Name
             </Text>
             <Text level='p' className='w-full font-semibold'>
```
Comment (imsks) [inline src/pages/webinar/[webinarSlug]/index.tsx:157]: remove semibold

Comment (imsks) [reply src/pages/webinar/[webinarSlug]/index.tsx:157]: also w-full

File: src/pages/webinar/[webinarSlug]/index.tsx [hunk 5]
```diff
@@ -152,10 +161,10 @@ const WebinarPage = ({
           </FlexContainer>
           <FlexContainer
             direction='col'
-            className='w-full md:w-[45%] max-w-[400px] '
+            className='w-full md:w-[45%] max-w-[400px]'
```
Comment (imsks) [inline src/pages/webinar/[webinarSlug]/index.tsx:163]: Remove abs values

File: src/pages/webinar/[webinarSlug]/index.tsx [hunk 6]
```diff
@@ -152,10 +161,10 @@ const WebinarPage = ({
...
             itemCenter={false}
           >
-            <Text level='label' className='pre-name  text-white font-semibold'>
+            <Text level='label' className='pre-name text-white font-semibold'>
```
Comment (imsks) [inline src/pages/webinar/[webinarSlug]/index.tsx:166]: here also

File: src/pages/webinar/[webinarSlug]/index.tsx [hunk 7]
```diff
@@ -225,7 +234,7 @@ const WebinarPage = ({
               <Image
                 alt='host-img'
                 src={host.imageUrl}
-                className='rounded-full w-[70px] md:w-20 lg:w-24 border-2 border-gray-950'
+                className='w-20 h-20 rounded-full border-2 border-zinc-900'
```
Comment (imsks) [inline src/pages/webinar/[webinarSlug]/index.tsx]: remove border classes

**Review by imsks — CHANGES_REQUESTED** [review]

File: src/pages/webinar/[webinarSlug]/index.tsx (hunk 1 above)
Comment (POBOROJO) [reply src/pages/webinar/[webinarSlug]/index.tsx:63]: made the file useCertificate.ts in hooks but could not implement it

**Pull request merged**